### ISSUE TITLE
SSR: document is not defined

### ISSUE THREAD

**Issue by abhipanda** [haltu/muuri#333]
During SSR getting document is not defined, it would be great if we can make this global check.

**Comment by niklasramo** [comment]
Yep, would be nice if Muuri played nice with SSR. I'm not fully sure what that would entail code-wise so if anyone wants to chime (e.g. PR) that would be awesome.

**Comment by abhipanda** [comment]
The idea is to use https://www.npmjs.com/package/global instead of directly using window / document.

Thanks
